### PR TITLE
Edit navbar

### PR DESCRIPTION
**Pull request by PolartJacques**
## Related Issues

Fixes #8

### CODE REVIEW

File: src/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -14,12 +14,22 @@ const Sidebar = () => {
...
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Menu menu={menu} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Contacts contacts={author.contacts} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Copyright copyright={copyright} />
+          </li>
```
Comment (AlexHff) [inline src/components/Sidebar/Sidebar.js]: Remove copyright from navbar

File: src/components/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -14,12 +14,22 @@ const Sidebar = () => {
...
+        <ul className={styles['sidebar__inner_list']}>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Logo img={logo} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Menu menu={menu} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Contacts contacts={author.contacts} />
+          </li>
```
Comment (AlexHff) [inline src/components/Sidebar/Sidebar.js]: This section can also be removed, we'll be putting all of this in the footer

**Review by AlexHff — CHANGES_REQUESTED** [review]
Sidebar folder and files should be renamed to reflect the fact that we have a Navbar instead.

Remove the grey border on the right of the screen. Navbar is buggy, when you scroll down and mouse over it, it moves the entire page. It also moves a little when you scroll down the first time.

Don't forget to check mobile usability.

File: src/components/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
+      <nav className={styles['sidebar__inner']}>
+        <ul className={styles['sidebar__inner_list']}>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Logo img={logo} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Menu menu={menu} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+          </li>
```
Comment (AlexHff) [inline src/components/Sidebar/Sidebar.js]: Why did you keep the empty list element?

**Review by AlexHff — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/Sidebar.js (hunk 3 above)
Comment (PolartJacques) [reply src/components/Sidebar/Sidebar.js]: easiest way to center the menu

Comment (AlexHff) [reply src/components/Sidebar/Sidebar.js]: I prefer the menu on the right hand side, just like on notion.so

**Comment by AlexHff** [comment]
! Rebased, dont forget to reset locally
The navbar should have the same margin on the right side as left.
image
Also, can you make it a little smaller in height?

**Review by AlexHff — APPROVED** [review]

**Comment by AlexHff** [comment]
Well done.

**Pull request merged**